packages/headless/src/api/search/search-api-client.ts

```diff
@@ -128,10 +128,12 @@ export class SearchAPIClient implements FacetSearchAPIClient {
   private searchAbortController: AbortController | null = null;
 
   async search(
-    req: SearchRequest
+    req: SearchRequest,
+    disableAbortWarning?: boolean
```

Comment:
In case we end up adding more options in the future, I would prefer to extract this new option into an interface. E.g.: `SearchOptions`.

Reply:
okie dokie